tensorflow/compiler/tf2xla/functionalize_cond.cc

```diff
@@ -235,14 +236,16 @@ StateMap::CondId StateMap::GetCondId(const StateMap::CondState& state) {
 }
 
 void StateMap::ResetCondId(const Node* node, StateMap::CondId id) {
-  if (node->id() < node_to_condid_map_.size())
+  const int64 node_to_condid_map_size = node_to_condid_map_.size(); 
+  if (node->id() < node_to_condid_map_size)
```

Comment:
Same as above. These locals are short lived, they are declared immediately before they are used so we don't need to be overly verbose.

Reply:
Don't forget about these ones.